### PR TITLE
GCI task for diambiguating text

### PR DESCRIPTION
**Pull request by namanrai**
i have added the file formatted_electricity.txt and added a line to manifest

### CODE REVIEW

**Comment by namanrai** [comment]
this is the pull request for the gci task

File: texts/formatted_electricity.txt [hunk 1]
```diff
@@ -0,0 +1,1626 @@
...
+"<charge>"
+	"charge" n sg
+;	"charge" vblex inf
+;	"charge" vblex pres
+;	"charge" vblex imp
+"<.>"
+	"." sent
+"<In>"
+	"in" adv
+	"in" pr
```
Comment (dolphingarlic) [inline texts/formatted_electricity.txt]: This hasn't been disambiguated

File: texts/formatted_electricity.txt [hunk 2]
```diff
@@ -0,0 +1,1626 @@
...
+"<early>"
+	"early" adv
+	"early" adj sint
```
Comment (dolphingarlic) [inline texts/formatted_electricity.txt]: This hasn't been disambiguated

File: texts/formatted_electricity.txt [hunk 3]
```diff
@@ -0,0 +1,1626 @@
...
+"<,>"
+	"," cm
+"<electricity>"
+	"electricity" n sg
+"<was>"
+;	"be" vbser past p1 sg
+	"be" vbser past p3 sg
+"<considered>"
+;	"consider" vblex pp
+	"consider" vblex past
```
Comment (dolphingarlic) [inline texts/formatted_electricity.txt]: This is a past participle

**Comment by namanrai** [comment]
The file has been in what format?
.TXT or .cg

**Pull request closed without merging**